### PR TITLE
Remove schedule native transfer

### PR DESCRIPTION
**Pull request by v9n**
This task is obsolute by schedule_dynamic_dispatch_tasks and passing the Balance extrinsic

### CODE REVIEW

File: pallets/automation-time/src/benchmarking.rs [hunk 1]
```diff
@@ -451,16 +383,11 @@ benchmarks! {
 		let weight_left = Weight::from_ref_time(500_000_000_000);
 		let mut task_ids = vec![];
 		let caller: T::AccountId = account("caller", 0, SEED);
-		let time = 10800;
-		let recipient: T::AccountId = account("to", 0, SEED);
-		let starting_multiplier: u32 = 20;
-		let transfer_amount = T::Currency::minimum_balance().saturating_mul(ED_MULTIPLIER.into());
-		let starting_amount = T::Currency::minimum_balance().saturating_mul(starting_multiplier.into());
-		T::Currency::deposit_creating(&caller, starting_amount.clone().saturating_mul(DEPOSIT_MULTIPLIER.into()));
+		let time1 = 10800;
```
Comment (chrisli30) [inline pallets/automation-time/src/benchmarking.rs:386]: remember to change it back to `executionTime`

File: pallets/automation-time/src/tests.rs [hunk 2]
```diff
@@ -1056,7 +1072,541 @@ mod run_dynamic_dispatch_action {
...
 // Weights to use for tests below
-// 20_000: run scheduled task (run_notify_task, run_
+// 20_000: run scheduled task (run_notify_task, run_native_transfer_task)
+// 10_000v: run per missed task (run_missed_tasks_many_found)
+// 10_000v: run per task not found in map (run_missed_tasks_many_missing, run_tasks_many_missing)
+// 50_000v: weight check for running 1 more task, current static v=1 (run_tasks_many_found)
+// 10_000: update task queue function overhead (update_task_queue_overhead)
+// 20_000: update task queue for scheduled tasks (update_scheduled_task_queue)
+// 20_000v: for each old time slot to missed tasks (append_to_missed_tasks)
+// 20_000: to move a single time slot to missed tasks (shift_missed_tasks)
```
Comment (chrisli30) [inline pallets/automation-time/src/tests.rs]: Yeah these comments and numbers are clear

**Review by chrisli30 — APPROVED** [review]
LGTM

**Review by imstar15 — CHANGES_REQUESTED** [review]
image

Please run the commands and fix to pass the two checks.

rustfmt
```
cargo fmt -- --check
```

Test

```
cargo test --workspace --all-features
```

File: pallets/automation-time/src/tests.rs [hunk 3]
```diff
@@ -217,75 +220,58 @@ fn schedule_not_enough_for_fees() {
...
-		}
+
+		Timestamp::set_timestamp(SCHEDULED_TIME * 1_000);
+		LastTimeSlot::<Test>::put((LAST_BLOCK_TIME, LAST_BLOCK_TIME));
+		System::reset_events();
+
+		let w = AutomationTime::trigger_tasks(Weight::from_ref_time(900_000_000));
+		let my_events = events();
+
+		let bob = AccountId32::new(BOB);
```
Comment (chrisli30) [inline pallets/automation-time/src/tests.rs:251]: Let’s not use name such as bob for a wallet, instead use a functional name. For example, if bob is the creator of the task, call it `creator` or `signer`.

This rule should apply to all tests, and we can update them whenever we work on other tests. Let me know what you think.

File: pallets/automation-time/src/tests.rs [hunk 4]
```diff
@@ -217,75 +220,58 @@ fn schedule_not_enough_for_fees() {
...
-						assert_eq!(task, expected_task);
-					},
-				},
-		}
+
+		Timestamp::set_timestamp(SCHEDULED_TIME * 1_000);
+		LastTimeSlot::<Test>::put((LAST_BLOCK_TIME, LAST_BLOCK_TIME));
+		System::reset_events();
+
+		let w = AutomationTime::trigger_tasks(Weight::from_ref_time(900_000_000));
```
Comment (chrisli30) [inline pallets/automation-time/src/tests.rs:248]: Let’s not use a name with a single letter like this. I assume this variable is never used below but still, please think of a meaningful name for this variable.

File: pallets/automation-time/src/mock.rs [hunk 5]
```diff
@@ -194,11 +194,37 @@ parameter_types! {
 	pub const MaxWeightPercentage: Perbill = Perbill::from_percent(10);
 	pub const UpdateQueueRatio: Perbill = Perbill::from_percent(50);
 	pub const ExecutionWeightFee: Balance = 12;
-	pub const MaxWeightPerSlot: u128 = 70_000_000;
+	pub const MaxWeightPerSlot: u128 = 700_000_000;
```
Comment (chrisli30) [inline pallets/automation-time/src/mock.rs:197]: Could you comment here, on the reason to 10x this value?

**Review by chrisli30 — COMMENTED** [review]
Looks good to me overall, but I have got several comments.

**Review by imstar15 — APPROVED** [review]
Looks good to me!

**Pull request merged**